spec/controllers/api/v1/api_controller_spec.rb

```diff
@@ -66,9 +66,9 @@ def index
       it { expect(subject).to eql(20) }
     end
 
-    context "30 per_page (limit is 25)" do
-      let(:per_page) { '30' }
-      it { expect(subject).to eql(25) }
+    context "30 per_page (limit is 50)" do
+      let(:per_page) { '60' }
+      it { expect(subject).to eql(50) }
```

Comment:
This spec is fine. Did you also know you can use `MAX_SEARCH_COUNT` here... depends on whether you think a spec should be declarative or narrative! I'm fine either way :)

Reply:
I thought about it.... I'd rather the spec dictates the implementation rather than the other way around 👍